### PR TITLE
chore(CVSB-14891): renamed some adr fields

### PR DESCRIPTION
**Pull request by rspasici**
PR for: https://jira.dvsacloud.uk/browse/CVSB-14891

### CODE REVIEW

File: src/app/technical-record/adr/adr-details/edit/__snapshots__/adr-details-edit.component.spec.ts.snap [hunk 1]
```diff
@@ -337,7 +337,7 @@ exports[`AdrDetailsEditComponent should create with initialized form controls 1`
                 class="govuk-label govuk-!-font-weight-bold"
                 for="adrTypeApprovalNo"
               >
-                 ADR type approval number 
+                 ADR type approval number(optional)
```
Comment (JonLazariniDD) [inline src/app/technical-record/adr/adr-details/edit/__snapshots__/adr-details-edit.component.spec.ts.snap:340]: should we have a space between the parenthesis and label?

File: src/app/technical-record/adr/declaration-seen/edit/__snapshots__/declaration-seen-edit.component.spec.ts.snap [hunk 2]
```diff
@@ -67,7 +67,7 @@ exports[`DeclarationSeenEditComponent should create with initialized form contro
                       class="govuk-label govuk-!-font-weight-bold"
                       for="brakeDeclarationIssuer"
                     >
-                       Issuer 
+                       Issuer(optional)(optional)
```
Comment (JonLazariniDD) [inline src/app/technical-record/adr/declaration-seen/edit/__snapshots__/declaration-seen-edit.component.spec.ts.snap:70]: we have optional twice, is that normal?

File: src/app/technical-record/adr/adr-details/edit/__snapshots__/adr-details-edit.component.spec.ts.snap (hunk 1 above)
Comment (alleypa) [reply src/app/technical-record/adr/adr-details/edit/__snapshots__/adr-details-edit.component.spec.ts.snap:340]: yes I agree. @rspasici  it should be like `olu (optional)` . Please lets update all where `optional` is used

File: src/app/technical-record/adr/adr-details/edit/adr-details-edit.component.html [hunk 3]
```diff
@@ -74,7 +74,7 @@ <h2 class="govuk-heading-m">ADR details</h2>
 
           <ng-template #compatibilityGroup>
             <div class="govuk-form-group govuk-!-margin-left-3">
-              <h4 class="govuk-heading-m">Compatibility group J</h4>
+              <h4 class="govuk-heading-m">Compatibility group J(optional)</h4>
```
Comment (alleypa) [inline src/app/technical-record/adr/adr-details/edit/adr-details-edit.component.html:77]: yes I agree with @JonLazariniDD . @rspasici it should be like olu (optional) . Please lets update all where optional is used

**Comment by alleypa** [comment]
Also add the jira ticket to your PR

**Comment by JonLazariniDD** [comment]
If we checked that we have all the fields required, please squash the commits so we have one node as merge commit. Thanks

**Comment by alleypa** [comment]
@rspasici please squash the commits on this branch to be ready for merging

**Pull request merged**